ui/plugins/controller.js

```diff
@@ -439,6 +439,7 @@ treeherder.controller('PluginCtrl', [
 
         var selectJobAndRender = function(job) {
             $scope.jobLoadedPromise = selectJob(job);
+            $( "#info-panel" ).addClass( "info-panel-slide" );
```

Comment:
I spoke with @wlach in channel, he agreed, so I will remove the paren space here, and I will open a separate bug to update them elsewhere in Treeherder.